core/frontend/src/PlanarClipMaskState.ts

```diff
@@ -53,14 +54,8 @@ export class PlanarClipMaskState {
 
     if (!this._tileTreeRefs) {
       this._tileTreeRefs = new Array<TileTreeReference>();
-      if (this.settings.modelIds) {
-        for (const modelId of this.settings.modelIds) {
-          const model = view.iModel.models.getLoaded(modelId);
-          assert(model !== undefined);   // Models should be loaded by RealityModelTileTree
-          if (model?.asGeometricModel)
-            this._tileTreeRefs.push(createMaskTreeReference(view, model.asGeometricModel));
-        }
-      }
+      if (this.settings.modelIds && view.isSpatialView())
```

Comment:
Here and in `getPlanarClipMaskSymbologyOverrides`, change `view` to a `SpatialViewState`.

In PlanarClassifier.ts, Ray has a bug where he is obtaining the alleged `ViewState3d` to pass to this the `PlanarClipMaskState`: `collectGraphics` does this:
```
    const viewState = context.viewingSpace.view as ViewState3d;
    if (undefined === viewState)
      return;
```

It's never going to `return` here because `as ViewState3d` just tells the compiler it definitely **is** a `ViewState3d`. Change it to check `view.isSpatialView()`.

Reply:
Done